Rakefile

```
@@ -40,7 +40,6 @@ namespace :xdr do
     require "pathname"
     require "xdrgen"
     require 'fileutils'
-    FileUtils.rm_f("xdr/xdr_generated.go")
```

❗ This looks unrelated and like an unnecessary change. Why is the change needed? I believe this is there to make sure that any xdr_generated.go we have at the end of a generate run is new and no silent error that doesn't result in a new file being written goes undetected.

---

It made development unpleasant because my IDE closed the buffer when the file was removed.

There is lo compelling reason to remove the file, since it's overwritten.

---

Ah interesting. We can solve both problems by replacing the `rm` with emptying the file. So the file never disappears, but it still behaves as before and is zeroed out?

---

I still don't see a compelling reason to reset the file.